cdci_data_analysis/flask_app/app.py

```diff
@@ -672,6 +677,38 @@ def load_frontend_fits_file_url():
         return make_response("fits_file_url arg not provided", 400)
 
 
+@app.route('/oauth_access_token_request', methods=['GET'])
+def oauth_access_token_request():
+    par_dic = request.values.to_dict()
+    logger.info('\033[32m===========================> oauth_access_token_request\033[0m')
+
+    client_id = par_dic.get('client_id', None)
+    code = par_dic.get('code', None)
+    redirect_uri = par_dic.get('redirect_uri', None)
+    client_secret = par_dic.get('client_secret', None)
```

Comment:
Given that this endpoint is meant to be used by the new client-only frontend, the client secret should never be there. 
If using the dispatcher as a backend helper in the authentication process anyway, the secret must be stored here.

Reply:
secrets is now extracted fro the dispatcher config